### ISSUE TITLE
Feature Request: Federated Access Control for SSO

### ISSUE THREAD

**Issue by ecliptik** [rancher/rancher#8809]
Feature Request to work with Single Sign-On Providers like Okta.

https://help.okta.com/en/prod/Content/Topics/Apps/Apps_Apps.htm

**Comment by kgeis** [comment]
FYI, blog article [Using Okta (and other SAML IdPs) with Rancher 2.0](https://mattslifebytes.com/2018/08/15/using-okta-and-other-saml-idps-with-rancher-2-0/). The author uses the SAML support in 2.0.7 and fakes Okta support by registering it as AD Federation Services (another SAML provider).

**Comment by ecliptik** [comment]
@kgeis thank you for the link. I came across it a few weeks ago and got Okta + Rancher 2.0 working by following his instructions (first time too!). I also followed up with @sullivanmatt who wrote it since we were seeing a few bugs and he was in touch with Rancher.

@superseb, Matt was working with @Oats87 , not sure what the status is but having Okta as a first-class SSO provider for Rancher would be excellent.
